src/utils/safeRename.js

```diff
@@ -0,0 +1,23 @@
+import { copyFile, rename, stat, unlink } from 'fs/promises';
+import fsExtra from 'fs-extra';
+
+export default async function safeRename(srcPath, destPath) {
```

Comment:
Please add a doc comment, describing how this is "safe"? For example, copy + remove means on some filesystems (… on Linux, though; not sure how it works on Windows & macOS), not flushing means none of this is safe :)

Reply:
Added.